### DISCUSSION TITLE
Duda test parte 2-2-inc

### DISCUSSION THREAD

**Discussion by isufan** [IIC2133-PUC/2024-1#57] [Tarea 0] [answered]
Hola,

Al probar mi código con el test de la parte 2-2-inc, me di cuenta de una situación que causa una pequeña diferencia de mi output con el output correcto. La línea 35 del input es PEDIDO-CONTAMINADO 3 13819, pero antes de que se ejecute ese comando, el pedido 13819 es entregado (línea 96 output). En el enunciado, para PEDIDO-CONTAMINADO se dice que el pedido contaminado no será de carácter entregado.

Quería saber si se trata de un error del test, o si para PEDIDO-CONTAMINADO también debo abarcar un caso donde se trata de eliminar un pedido entregado, ya que hasta ahora asumía que ningún pedido iba a estar entregado.

**Comment by florencialira** [comment]
Me pasó lo mismo con el pedido 12091. En entregar pedidos se entregan todos y después sale lo del pedido contaminado (línea 31-33 del input) por lo que no se encuentra, y en el output dice que fue eliminado pero en verdad debería decir no encontrado.

**Comment by isufan** [comment]
Ayer también me di cuenta que en los tests parte-2-3-inc y parte-2-4-inc ocurre la misma situación. Les agradecería si se puede revisar y aclarar.

**Comment by PabloNavarroY** [comment, marked as answer]
No sabemos cómo esa línea llegó al enunciado.

![IMG_20240327_181659_296.jpg](https://github.com/IIC2133-PUC/2024-1/assets/155672565/3b21b482-f032-4b67-9946-fbfbdc350026)

Hagan como que el enunciado dice lo de la imagen.

Perdón por la confusión.
